assets/css/_route_ladder.scss

```diff
@@ -65,22 +65,16 @@
   .c-route-ladder__alert-icon {
     display: flex;
     align-items: center;
-    width: 1.6rem;
 
     cursor: pointer;
 
     svg {
-      height: 1.6rem;
-      width: 1.6rem;
+      height: 1rem;
+      width: 1.125rem;
     }
   }
 
-  .c-route-ladder__route-pill {
-    grid-column: 2/3;
-  }
-
   .c-route-ladder__close-button-container {
-    grid-column: 3/4;
```

Comment:
question: why was this deleted? it seems that the grid is still intact, so I'd assume that the only thing this changes is making the grid column it occupies implicit from DOM order instead of explicitly defined here.

Reply:
> And all I really need the grid to do here is allow the center cell to take the space it needs, and the first and third cell to have the same width. Is this the right way to do that?

That is the correct way to do that. For example, just so that we can precisely talk about what's happening, I'll give the columns names based on their numbers
```
1: 1fr
2: auto
3: 1fr
```

If you had this on a parent element (named `grid-container`), with 3 children without any styling
```html
<grid-container>
	<child-1/>
	<child-2/>
	<child-3/>
</grid-container>
```
and with this grid defined on the `grid-container`
```css
grid-container {
	/* inline or "regular" grid, doesn't matter */
	/* display: inline-grid; */
	/* display: grid; */
	grid-template-columns: 1fr auto 1fr;
}
```

`child-1` would be in grid column `1`, `child-2` would be in column `2`, and `child-3` in `3`.

If you rearranged them
```html
<grid-container>
	<child-1/>
	<child-3/>
	<child-2/>
</grid-container>
```
`child-1` would be in `1`, `child-2` would be in column `3`, and `child-3` would be in column `2`.

If instead these children had styling telling them which column to be in, 
```css
child-1 {
	grid-column: 1/2;
}

child-2 {
	grid-column: 2/3;
}

child-3 {
	grid-column: 3/4;
}
```

Then the style is now independent of the DOM order, which is good for two reasons.
A: It allows us to rearrange the order of things in the accessibility tree, and 

B: we don't need to worry that somehow accidentally the elements will get rearranged without noticing and the style then breaks because, from the first re-arranging example, `child-3` would not be stretched to `auto` (with the route ladders, this means the close button would be stretched).

A is beneficial because to a screen reader, right now reading the route ladder headers would read as "options, route <name>, remove" but honestly, it doesn't really make sense to hear that there are options, before you know what route you're operating on and the order _should_ be "route <name>, options, remove" which would have a example DOM order like this
```html
<grid-container>
	<child-2/> <!-- route -->
	<child-1/> <!-- options -->
	<child-3/> <!-- close -->
</grid-container>
```

So all-in-all, I feel more comfortable being explicit with grid styling then I do about having grid styling be implicit by DOM order.

Reply:
Ohhhhhh interesting!! My accessibility expertise is stale / nonexistent, so I didn't realize the accessibility tree could be ordered differently from the DOM. The example you provide in regards to the route ladder header readout was the biggest "ohhhh" realization for me. Thanks, I'll tweak it

Reply:
Just noting this from DMs, I wrote all that to point out what we _could_ do in the future by keeping the `grid-colunm` property explicit, and what happens if it's not explicit. Just to be clear, I'm **not** advocating for changing the DOM order _in this PR_.